src/common/options/crimson.yaml.in

```diff
@@ -40,6 +40,11 @@ options:
   desc: Total size to use for SegmentManager block file if created
   level: dev
   default: 50_G
+- name: seastore_shard_device_size
+  type: size
+  desc: Total size to use for SegmentManager block file = max(seastore_device_size, seastore_shard_device_size * shard_num)
+  level: dev
+  default: 5_G
```

Comment:
currently each shard size is seastore_device_size/seastar::smp::count, but when seastore_device_size = 50G(default setting), smp::cout >28, seastore can't startup. so you have to enlarge seastore_device_size , when set seastore_device_size  =100 G, smp::cout =64, seastore also can't startup, I just add a shard_device_size, when seastore_device_size > shard_device_size * smp::cout , use seastore_device_size  as total size, otherwise use shard_device_size * smp::cout  as total size to guarantee seastore startup, then you needn't always adjust seastore_device_size  according to cores increasing since no obvious message tell you to resetting  seastore_device_size to a proper size when seastore not startup.

Reply:
I think there are better ways other than adding an additional configuration for the device size, including but not limited to:
* Print explicit error message and ceph_abort() if the configuration is not appropriate.
* Adjust vstart script to adjust the seastore device size more easily for all kinds of seastore tests.
* Increase the default seastore-device-size if that's not sufficient **normally**, smp==64 doesn't look to me like a normal/real-world use case, it looks to be a special-purpose evaluation that needs to be configurred specially.

Reply:
@cyx1231st add --seastore-device-size setting in vstart.